stl/inc/bit

```diff
@@ -253,7 +252,7 @@ _NODISCARD int _Checked_x86_x64_countl_zero(const _Ty _Val) noexcept {
 #ifdef __AVX2__
     return _Countl_zero_lzcnt(_Val);
 #else // __AVX2__
-    const bool _Definitely_have_lzcnt = __isa_available >= __ISA_AVAILABLE_AVX2;
+    const bool _Definitely_have_lzcnt = __isa_available >= 5; // __ISA_AVAILABLE_AVX2

```

Comment:
I'd feel more comfortable with the duplication if we gave this a symbolic name and had test coverage to assert that it's equal to `__ISA_AVAILABLE_AVX2`.